bin/k8s-deploy

```diff
@@ -152,6 +152,24 @@ do
     echo "Updating ${DEPLOYMENT_FILE}-${CI_SHA1}"
     DEPLOYMENT_FILE=${DEPLOYMENT_FILE}-${CI_SHA1}
   fi
+  if [[ $( grep -i replicas $DEPLOYMENT_FILE | awk -F: {'print $2'} | grep -i hpa ) ]]
```

Comment:
This can avoid using 3 commands by using awk to match the regex as well. Depending on how precise we want to be with regex'ing a yaml file this can be something like: 

```
[[ -n $( awk -F: '/replicas[[:space:]]*:[[:space:]]*+hpa/' $DEPLOYMENT_FILE )
```

Not critical, but didn't know at first glance what this was looking for.